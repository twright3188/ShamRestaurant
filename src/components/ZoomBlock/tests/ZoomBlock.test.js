import React from 'react';
import renderer from 'react-test-renderer';
import { ZoomBlock } from '../index';

it('zoom block render', () => {
    const tree = renderer.create(<ZoomBlock isPressed={false} onPress={() => {}} value='test' />).toJSON();
    expect(tree).toMatchSnapshot();
});