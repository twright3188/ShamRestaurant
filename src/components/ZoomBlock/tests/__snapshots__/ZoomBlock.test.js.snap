// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`zoom block render 1`] = `
<View
  accessible={true}
  clickable={true}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "borderColor": "#399AE5",
      "borderRadius": 16,
      "borderWidth": 1,
      "flexDirection": "row",
      "opacity": 1,
      "paddingHorizontal": 8,
      "paddingVertical": 4,
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "flexWrap": "wrap",
          "fontFamily": "Helvetica",
          "fontWeight": "400",
        },
        Object {
          "color": "#399AE5",
          "fontSize": 11,
          "lineHeight": 16.5,
        },
        Object {},
      ]
    }
  >
    test
  </Text>
</View>
`;
